lib/policies/proxy/round-robin.js

```diff
@@ -0,0 +1,30 @@
+const logger = require('../../logger').gateway;
+
+module.exports = class RoundRobin {
+  constructor (proxy, endpoints) {
```

Comment:
For this specific strategy I believe proxy in Strategy is not needed. 
having `nextTarget` interface should be enough 
same for 'sticky' type

is proxy here for future least-conn type to count requests\responses?